### PR TITLE
Review 190422

### PR DESCRIPTION
**Pull request by zeue**
This PR changes the handbook by implementing some corrections to terminology as well as changes to shinies, the reserves system, and added a new rule.

### CODE REVIEW

File: docs/the-unit/getting-started.md [hunk 1]
```diff
@@ -8,23 +8,29 @@ Join the [Discord](https://uagpmc.com/discord) and ask for a recruiter or messag
 
 ### 2a. When are our operations?
 
-Our operation times are **Saturday and Sunday at 18:00 UTC** check on the bottom of our [website](https://uagpmc.com/) for a timer till our next operation.
+Our operation times are **Saturday and Sunday at 17:00 UTC** check on the bottom of our [website](https://uagpmc.com/) for a timer till our next operation.
 
 ### 2b. Can't Attend a Session?
 
-If you cant attend a session during your time as a recruit please contact one of the Big Boys on discord or steam and let them know 24 hours in advance. Note that currently Zyrtex handles attendance for all members and recruits.
+If you cant attend a session during your time as a shiny please contact one of the "helpers" on discord or steam and let them know 24 hours in advance. Note that currently Zyrtex handles attendance for all members and shinies and should be your first-contact for these matters.
```
Comment (MemeDream2013) [inline docs/the-unit/getting-started.md]: It triggers me there is no comma after "Zyrtex handles attendance for all members and shinies" but I could be a dumb dumb

File: docs/the-unit/reserves-system.md [hunk 2]
```diff
@@ -29,12 +29,12 @@ The process of reincorporation is the following:
...
 
-The only requirements of the reserves system is that you must be a full member, **recruits do not get to apply!**
+The only requirements of the reserves system is that you must be a full member, **shinies do not get to apply for reserves!**
 
 ### 3b. Restrictions
 
 The following restrictions are applied to all reserves who join a session:
 
-- Reserves are not given a choice of role in-game, and must fill less-desired roles if they are requested to.
+- Reserves choose their roles before shinies but after all other group members, and must fill less-desired roles if they are requested to by the section commander.
```
Comment (Zyrtex) [inline docs/the-unit/reserves-system.md]: reserves choose their roles before shinies but after group members.
Conflict with "getting started" line 29 :
Shinies choose their roles before the rest of the group

Comment (MemeDream2013) [inline docs/the-unit/reserves-system.md]: How do Reserves pick before shinies but after the other members when shinies pick before the members

File: docs/the-unit/getting-started.md (hunk 1 above)
Comment (Zyrtex) [reply docs/the-unit/getting-started.md]: i call rule 5 on this

Comment (Radrakin) [inline docs/the-unit/getting-started.md]:
```suggestion
If you cant attend a session during your time as a shiny please contact one of the "helpers" on discord or steam and let them know 24 hours in advance. Note that currently Zyrtex handles attendance for all members and shinies; He should be your first-contact for these matters.
```

File: docs/the-unit/reserves-system.md [hunk 3]
```diff
@@ -29,12 +29,12 @@ The process of reincorporation is the following:
 
 ### 3a. Requirements
 
-The only requirements of the reserves system is that you must be a full member, **recruits do not get to apply!**
+The only requirements of the reserves system is that you must be a full member, **shinies do not get to apply for reserves!**
```
Comment (Radrakin) [inline docs/the-unit/reserves-system.md]:
```suggestion
The only requirements of the reserves system is that you must be a full member, **shinies can not get to apply for reserves!**
```

File: docs/the-unit/reserves-system.md (hunk 2 above)
Comment (zeue) [reply docs/the-unit/reserves-system.md]: Resolved in cce7982d74405ef182997dcfc6cdaf04075bf94a

File: docs/the-unit/getting-started.md (hunk 1 above)
Comment (zeue) [reply docs/the-unit/getting-started.md]: I agree, needless change

File: docs/the-unit/reserves-system.md (hunk 2 above)
Comment (zeue) [reply docs/the-unit/reserves-system.md]: Already resolved in cce7982d74405ef182997dcfc6cdaf04075bf94a

**Comment by zeue** [comment]
> I agree with Zyrtex ([uagpmc/handbook/pull/1/files#r277325863](https://github.com/uagpmc/handbook/pull/1/files#r277325863))

Resolved with cce7982d74405ef182997dcfc6cdaf04075bf94a

File: docs/the-unit/reserves-system.md (hunk 3 above)
Comment (zeue) [reply docs/the-unit/reserves-system.md]: What about this? @Radrakin 

```suggestion
The only requirement of the reserves system is that you must be a full member, **shinies can't apply for reserves!**
```

Comment (Radrakin) [reply docs/the-unit/reserves-system.md]: Yep.

**Pull request merged**